static-build/qemu-virtiofs/Dockerfile

```diff
@@ -49,23 +49,17 @@ RUN cd  .. && git clone "${QEMU_VIRTIOFS_REPO}" qemu-virtiofs
 RUN git checkout "${QEMU_VIRTIOFS_TAG}"
 
 ADD scripts/configure-hypervisor.sh /root/configure-hypervisor.sh
-ADD qemu/patches/ /root/kata_qemu_patches
+ADD qemu /root/kata_qemu
 
 # Apply experimental specific patches
 # Patches to quick fix virtiofs fork
-ENV VIRTIOFS_PATCHES_DIR=/root/kata_qemu_patches/${QEMU_VIRTIOFS_TAG}/
-RUN find "${VIRTIOFS_PATCHES_DIR}" -name '*.patch' -type f |sort -t- -k1,1n > patches_virtiofs
-RUN echo "Patches to apply for virtiofs fixes:"
-RUN cat patches_virtiofs
-RUN [ ! -s patches_virtiofs ] || git apply $(cat patches_virtiofs)
-
-RUN cat VERSION | awk 'BEGIN{FS=OFS="."}{print $1 "." $2 ".x"}' > stable_branch
-RUN echo "root/kata_qemu_patches/$(cat stable_branch)/" > patches_qemu_dir
-RUN echo "patches dir $(cat patches_qemu_dir)"
-RUN find "$(cat patches_qemu_dir)" -name '*.patch' -type f |sort -t- -k1,1n > patches_qemu
-RUN echo "Patches to apply for qemu:"
-RUN cat patches_qemu
-RUN [ ! -s patches_qemu ] || git apply $(cat patches_qemu )
+ENV VIRTIOFS_PATCHES_DIR=/root/kata_qemu/patches/${QEMU_VIRTIOFS_TAG}/
+RUN if [ -d ${VIRTIOFS_PATCHES_DIR} ]; then \
+	echo "Patches to apply for virtiofs fixes:"; \
+	for patch in $(find "${VIRTIOFS_PATCHES_DIR}" -name '*.patch' -type f |sort -t- -k1,1n); do \
+		git apply $patch; \
+	done;fi
```

Comment:
Please change this block to:

- Add the `sort` functionality to `apply_patches.sh`.
- Call `apply_patches.sh $VIRTIOFS_PATCHES_DIR`.